charts/newrelic-k8s-metrics-adapter/tests/configmap_test.yaml

```
@@ -0,0 +1,92 @@
+suite: test configmap region helper and externalMetrics
+templates:
+  - templates/configmap.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+tests:
+  - it: has the correct region when defined in local values
+    set:
+      personalAPIKey: 21321
+      config:
+        accountID: 111
+        region: A-REGION
```

If they are needed you can put them in the header so you don't bother about them and have to repeat them.
```suggestion
release:
  name: my-release
  namespace: my-namespace
set:
  personalAPIKey: 21321
  config:
    accountID: 111
tests:
  - it: has the correct region when defined in local values
    set:
      config:
        region: A-REGION
```

---

It's not working, actually in the tests of the common library they are not doing anything.